app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharecterView.kt

```diff
@@ -50,4 +67,44 @@ class CharecterView(activity: MainActivity) {
         activityRef.get()!!.progressBar.visibility = View.VISIBLE
 
     }
+
+
+
+    fun showCharacterInformation(character: Character) {
+        val dialogFragment = CharacterDialogFragment()
+        val args = Bundle()
+        args.putParcelable(EXTRA_CHARACTER, character)
+        dialogFragment.arguments = args
+        dialogFragment.show(activityRef.get()?.supportFragmentManager, "txn_tag")
+    }
+
+    class CharacterDialogFragment() : DialogFragment() {
```

Comment:
put this in a separated class pls

Reply:
Hernan, please move this, into a new class with his MVP implementation, to separate the functionality.

Reply:
Done